panel/tests/widgets/test_tqdm.py

```diff
@@ -19,6 +19,18 @@ def test_tqdm():
     assert isinstance(tqdm.layout, pn.Row)
 
 
```

Comment:
It would be very nice if an example could be included in the TQDM reference docs?

At least please include a simple minimum, reproducible Panel app example in this PR, that shows how this can be used in a Panel app. Then its easy to later include in the docs if needed.

Thanks.

Reply:
Is there a place in the docs I should edit as part of this PR? I'm looking for a path in the repo.

The other option is to add an example to the PR description. I'm unclear on what form the example should be.